tools/dev/changelog.py

```diff
@@ -6,12 +6,13 @@
 import json
 
 from collections import defaultdict
+from distutils import version
```

Comment:
Since this is new code, may be worth dodging a deprecation warning in 3.10+

```suggestion
from packaging.version import parse as parse_version
```